bin/content-push

```diff
@@ -14,12 +14,28 @@ def ask_for_name
   name
 end
 
+def disallow_case_change
+  `git status --porcelain --untracked-files=all #{CONTENT_PATHS}`.split("\n").each do |line|
+    if (%r{^R  ([^ ]+) -> ([^ ]+)}.match(line) || %r{^R  ("[^"]+") -> ("[^"]+")}.match(line)) && $1.casecmp($2) == 0
```

Comment:
Good point. Upon further investigation, it appears this format is guaranteed to never change according to `git help status`:

>The porcelain format is similar to the short format, but is guaranteed not to change in a
>backwards-incompatible way between Git versions or based on user configuration. This makes it ideal
>for parsing by scripts.